src/shm_main.c

```diff
@@ -43,6 +44,37 @@ struct shm_item {
     char *name;
 };
 
+/**
+ * @brief Item holding information about active connections owned by this process.
+ */
+typedef struct _conn_list_entry {
+    struct _conn_list_entry *_next;
+    sr_cid_t cid;
+    int cid_fd;
+} conn_list_entry;
```

Comment:
Now that I think about, I would prefer the type to have `sr_` prefix, but just the `conn_list_entry`, the struct is irrelevant.

Reply:
Done